model_compression_toolkit/target_platform_capabilities/tpc_models/get_target_platform_capabilities.py

```diff
@@ -16,17 +16,17 @@
 from model_compression_toolkit.target_platform_capabilities.target_platform import TargetPlatformCapabilities
 
 from model_compression_toolkit.target_platform_capabilities.tpc_models.imx500_tpc.target_platform_capabilities import \
-    tpc_dict as imx500_tpc_dict
+    get_tpc_dict_by_fw as imx500_tpc
```

Comment:
the "as" name is confusing, it is a function not an initialized TPC, maybe add "get_" at the beggining